### ISSUE TITLE
Missing a dependency

### ISSUE THREAD

**Issue by graysky2** [ZivaVatra/flac2all#43]
I am [packaging](https://aur.archlinux.org/packages/flac2all/) this for Arch Linux.  I took the dependencies from your README but perhaps I missed one?  Any thoughts are welcomed.

```
% flac2all -h
Traceback (most recent call last):
  File "/usr/bin/flac2all", line 11, in <module>
    load_entry_point('flac2all==4.3', 'console_scripts', 'flac2all')()
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2852, in load_entry_point
    return ep.load()
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2443, in load
    return self.resolve()
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2449, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3.8/site-packages/flac2all_pkg/__init__.py", line 32, in <module>
    from shell import shell
ModuleNotFoundError: No module named 'shell'
```

**Comment by fwillo** [comment]
Hey

I experienced the same issue today. Commit https://github.com/ZivaVatra/flac2all/commit/dfe6d201d57866905dabf6bbcae4d396e4e4d130 says that Python 3 messed up the relative imports. The shell and the config imports are submodules, which were imported as the [diff](https://github.com/ZivaVatra/flac2all/commit/dfe6d201d57866905dabf6bbcae4d396e4e4d130) shows. Unfortunately tag v5.0 is the latest one. I fixed your PKGBUILD (temporarily) for now with a direct use of the commit instead of the tag.
```
--- ../flac2all/PKGBUILD	2020-01-22 20:47:49.408513771 +0100
+++ PKGBUILD	2020-01-22 20:47:19.321042657 +0100
@@ -18,8 +18,8 @@
 'python-pyzmq: for clustering'
 'vorbis-tools: for ogg support'
 )
-source=("$pkgname-$pkgver.tar.gz::https://github.com/ZivaVatra/$pkgname/archive/v$pkgver.tar.gz")
-sha256sums=('d18a01653eed4d497cc1fe0393fecfaf6b6eab5dac9bd5599d1e83231b76c22d')
+source=("$pkgname-$pkgver::git+https://github.com/ZivaVatra/flac2all.git#tag=20c96df4d88130dc19caab7ba93d962c6e64fee6")
+sha256sums=('SKIP')
 
 prepare() {
   cd "$pkgname-$pkgver"
```
Maybe this helps for now. But a new tag should be made for the import fixes due to their importance, I think.

Best,
fwillo

**Comment by graysky2** [comment]
@fwillo - Fixed in [5.0-2](https://aur.archlinux.org/cgit/aur.git/commit/?h=flac2all&id=995345d2c74cf2470c8c2dbe54d999ccbd8da24f) thanks.

**Comment by ZivaVatra** [comment]
Yes, I have been bitten by this too while trying to package flac2all for pip3. I have done the commit to correct the issue, and am currently testing. Once the pip3 package builds and works, I will tag a new release version.
